topgun/k8s/kubernetes_creds_mgmt_test.go

```diff
@@ -88,48 +87,106 @@ var _ = Describe("Kubernetes credential management", func() {
 
 			fly.Run("unpause-pipeline", "-p", "pipeline")
 
-		})
-
-		var runsBuildWithCredentialsResolved = func() {
 			session := fly.Start("trigger-job", "-j", "pipeline/unit", "-w")
 			Wait(session)
 
+			By("seeing the credentials were resolved by concourse")
 			Expect(string(session.Out.Contents())).To(ContainSubstring("bar"))
 			Expect(string(session.Out.Contents())).To(ContainSubstring("zaz"))
 		}
 
-		Context("using the default namespace created by the chart", func() {
-			It("succeeds", runsBuildWithCredentialsResolved)
-		})
+		var runGetsCachedCredentials = func(normalSecret string, specialKeySecret string) {
```

Comment:
Nice! It's pretty cool how much readable it becomes when looking at the context and specs descriptions once everything comes down to those methods.

I was wondering if we could get this out of the `context` block though, just like `createCredentialSecret` is separated. Wdyt?

thanks!

Reply:
Moved, looks better 👍